src/pheval/utils/utils.py

```diff
@@ -0,0 +1,149 @@
+        max_num = dataframe[col].max()
+        dataframe[col] = dataframe[col].apply(rand, args=(min_num, max_num, scramble_factor))
+    return dataframe
+
+
+def semsim_convert(input: Path, output: Path, subject_prefix: str, object_prefix: str, format: str):
+    if format == "exomiserdb":
+        return semsimconvert_exomiserdb(input, output, subject_prefix, object_prefix)
+    raise ValueError("Invalid format")
+
+
+def semsimconvert_exomiserdb(input: Path, output: Path, subject_prefix: str, object_prefix: str):
+    """convert semsim profile to an exomiser specie mapping schema
+    Example of a mapping schema:
+
+    MAPPING_ID;HP_ID;HP_TERM;MP_ID;MP_TERM;SIMJ;IC;SCORE;LCS_ID;LCS_TERM
+
+    To generate the example above, the parameter subject_prefix needs to be "HP" and object_prefix needs to be "MP"
+
+    Args:
+        input (Path): Path to the semsim profile
+        output (Path): Path where sql file will be written
+        subject_prefix: (str): mapping subject prefix (e.g. HP)
+        object_prefx (str): mapping object prefix (e.g. MP)
+    """
+    input_data = pd.read_csv(input, sep="\t")
+    input_data = input_data.rename(
+        columns={
+            "subject_id": f"{subject_prefix}_ID",
+            "subject_label": f"{subject_prefix}_TERM",
+            "object_id": f"{object_prefix}_ID",
+            "object_label": f"{object_prefix}_TERM",
+            "jaccard_similarity": "SIMJ",
+            "ancestor_information_content": "IC",
+            "phenodigm_score": "SCORE",
+            "ancestor_id": "LCS_ID",
+            "ancestor_label": "LCS_TERM",
+        }
+    )[
+        [
+            f"{subject_prefix}_ID",
+            f"{subject_prefix}_TERM",
+            f"{object_prefix}_ID",
+            f"{object_prefix}_TERM",
+            "SIMJ",
+            "IC",
+            "SCORE",
+            "LCS_ID",
+            "LCS_TERM",
+        ]
+    ]
+    input_data.insert(0, "MAPPING_ID", None)
+    input_data["SCORE"].replace("None", "NULL", inplace=True)
+    input_data.to_csv(output, index=False, sep="\t")
+    semsim2h2(input_data, f"{output}", subject_prefix, object_prefix)
+
+
+def semsim2h2(input: Path, output: Path, subject_prefix: str, object_prefix: str):
+    """This function converts the exomiser mapping table to sql format.
+    Args:
+        input (Path): Path to the semsim profile
+        output (Path): Path where sql file will be written
+        subject_prefix: (str): mapping subject prefix (e.g. HP)
+        object_prefx (str): mapping object prefix (e.g. MP)
+    """
+    with open(output, "w") as file:
+        insert = f"""
+TRUNCATE TABLE EXOMISER.{subject_prefix}_{object_prefix}_MAPPINGS;
+INSERT INTO EXOMISER.{subject_prefix}_{object_prefix}_MAPPINGS
+(MAPPING_ID, {subject_prefix}_ID, {subject_prefix}_TERM, {object_prefix}_ID, {object_prefix}_TERM, SIMJ, IC, SCORE, LCS_ID, LCS_TERM)
+VALUES
+"""  # noqa
+        file.write(insert)
+        for idx, data in input.iterrows():
```

Comment:
Its strange qc does not catch this, but `input` is a Path according to method signature, and your you so iterrows.